plugins/linux-capture/pipewire.c

```diff
@@ -567,6 +574,14 @@ static void on_param_changed_cb(void *user_data, uint32_t id,
 
 	spa_format_video_raw_parse(param, &obs_pw->format.info.raw);
 
+	// Announce support for DMA-BUF only if negotiation results in a modifier
+	if (has_pw_version(&obs_pw->server_version, 0, 3, 24) &&
+	    has_pw_version(&obs_pw->library_version, 0, 3, 24) &&
+	    has_pw_version(&obs_pw->header_version, 0, 3, 24) &&
+	    !has_pw_version(&obs_pw->server_version, 0, 3, 40)) {
```

Comment:
Please remove this last `!has_pw_version(&obs_pw->server_version, 0, 3, 40)` - we'll deal with 0.3.40 in your other PR, but meanwhile, let's make this PR be only about version < 0.3.24 → memptr.